scripts/test_validations.py

```diff
@@ -0,0 +1,44 @@
+from nose.tools import eq_
```

Comment:
What that means is that you should use simple `assert`s instead of the `eq_` helper.

Py.test has a mecanism that provides you as much (and often much more) information about the failure when using `assert`